### ISSUE TITLE
Program stuck on 'Checking license'

### ISSUE THREAD

**Issue by alishahlakhani** [open-wa/wa-automate-nodejs#485]
**Describe the bug**
Stuck on **'Checking License'**.

**Steps to Reproduce**
Steps to reproduce the behaviour:
1. Run the code with subscribed license key and the system either gets stuck on 'Checking license' or passes and does nothing.

**create() code**
This is the code you use to create the client. e.g
```javascript
client = await sulla.create('app', {
        licenseKey: "..."
});
```

**Expected behavior**
The app should successfully connect with client and proceed with next callback

**DEBUG INFO**
This is the info printed to the console when you start your app. It should look like this
```
✓ Initializing WA
WA_VERSION: '2.2021.4',
PAGE_UA: 'WhatsApp/2.2019.8 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36',
WA_AUTOMATE_VERSION: '1.9.47',
BROWSER_VERSION: 'HeadlessChrome/81.0.4044.138',
✓ Authenticated
✓ WAPI Reinjected
✓ Client is ready
✓ Passed Integrity Test
```

**Screenshots**
<img width="1240" alt="Screenshot 2020-05-25 at 11 42 39 PM" src="https://user-images.githubusercontent.com/12266036/82827341-6d7d9b80-9ee1-11ea-9077-bf38fe89dd77.png">

**Comment by alishahlakhani** [comment]
After debugging, it seems like something is wrong with my license key. when I remove it my code works just fine. Can you help me check what's wrong with my key?

**Comment by smashah** [comment]
sorry about this @alishahlakhani I need to move the license server off glitch. Can you dm me in the discord to debug asap (@ SHAH)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.49](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.48...1.9.49) (2020-05-26)

- Release 1.9.49 [`f64047b`](https://github.com/open-wa/wa-automate-nodejs/commit/f64047b6f0783fb2289628e0c3e4acb1022d8a7b)
- **IMPORTANT** fix: frozen on license check #485 [`962dd9c`](https://github.com/open-wa/wa-automate-nodejs/commit/962dd9cbd18318e0343e7ad5ea445c9da5546062)

**Comment by alishahlakhani** [comment]
Thanks it worked!
